docs/console.html

```diff
@@ -0,0 +1,804 @@
+                    <figcaption>Example code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+function AddResult(name, result) {
+    console.timeStamp("Adding result");
+    var text = name + ': ' + result;
+    var results = document.getElementById("results");
+    results.innerHTML += (text + "&lt;br>");
+}
+</code></pre>
+                </figure>
+                <figure>
+                    <img src="console-files/timestamp2.png" alt="timestamps in the timeline" />
+                    <figcaption>
+                        Timestamps in the timeline.
+                    </figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="setting-breakpoints-in-javascript">Setting breakpoints in JavaScript</h3>
+            <p>
+                The <code><a href="console-api#debugger">debugger()</a></code> statement will begin a debugging session.
+                This is equivalent to setting a breakpoint at this line in the script.
+            </p>
+
+            <figure>
+                <figcaption>The debug session will begin as <code>brightness()</code> is called.</figcaption>
+                <figure>
+                    <figcaption>Example code:</figcaption>
+                    <pre class="prettify"><code class="language-javascript">
+brightness : function() {
+    debugger;
+    var r = Math.floor(this.red*255);
+    var g = Math.floor(this.green*255);
+    var b = Math.floor(this.blue*255);
+    return (r * 77 + g * 150 + b * 29) >> 8;
+}
+</code></pre>
+                </figure>
+                <figure>
+                    <figcaption>Example output:</figcaption>
+                    <img src="console-files/debugger.png" alt="Debugger activated" />
+                </figure>
+            </figure>
+        </section>
+</div>
+
+<div class="collapsible">
+    <h2 id="using-the-command-line-api">Using the Command Line API</h2>
+    <p>
+        The console is more powerful than a simple log output directory.
+        It also is a full terminal prompt in the scope of the current web page.
+        The <a href="commandline-api">Command Line API</a> offers the following features:
+    </p>
+    <ul>
+        <li>Convenience functions for selecting DOM elements</li>
+        <li>Methods for controlling the CPU profiler</li>
+        <li>Aliases for a number of Console API methods</li>
+        <li>Monitoring events</li>
+        <li>View event listeners registered on objects</li>
+    </ul>
+
+    <section>
+        <h3 id="evaluating-expressions">Evaluating expressions</h3>
+        <p>
+            The console will evaluate any JavaScript expression you provide when pressing <kbd class="kbd">Enter</kbd>.
+            It provides auto-completion and tab-completion.
+            As an expression is typed property names are suggested.
+            If there are multiple matches <kbd class="kbd">Tab</kbd> will cycle through them.
```

Comment:
Hmm... it was working for me on Windows iirc... Linux it isn't. I'll update with arrows since I know that works in both places I can test. Maybe something was re-factored here and tabbing was removed.